jsplot/src/main/java/tech/tablesaw/plotly/Utils.java

```diff
@@ -53,4 +56,53 @@ public static String dataAsString(double[][] data) {
     builder.append("]");
     return builder.toString();
   }
+
+  /**
+   * Returns a list of column without missing value
+   *
+   * @param columns a list of columns that may contain missing value
```

Comment:
Please change the doc to say array where the argument or return value is an array rather than a List